background.js

```diff
@@ -707,6 +707,45 @@ browser.compose.onBeforeSend.addListener(tab => {
   }
 });
 
+browser.runtime.onMessageExternal.addListener(externalListener);
```

Comment:
Also stylistic, but for consistency can this change to an arrow function?

Reply:
I will change it, but just FYI the reason why I used a named function rather than an arrow function is because the arrow function is indented more than the named function would be, and I am an old-school developer who still cares about lines being less than 80 characters. ;-)